code/modules/cargo/delivery/backpack.dm

```diff
@@ -0,0 +1,98 @@
+/obj/item/cargo_backpack
+	name = "cargo pack"
+	desc = "A robust set of rigs and buckles that allows the wearer to carry two additional Orion Express delivery packages on their back."
+	desc_info = "To load packages onto your back, equip this item on the back slot, then click on it with a package in-hand. To unload a package, click on this item with an empty hand."
+	icon = 'icons/obj/orion_delivery.dmi'
+	icon_state = "package_pack"
+	item_state = "package_pack"
+	contained_sprite = TRUE
+	w_class = ITEMSIZE_HUGE
+	slot_flags = SLOT_BACK
+	drop_sound = 'sound/items/drop/backpack.ogg'
+	pickup_sound = 'sound/items/pickup/backpack.ogg'
+
+	var/list/contained_packages
+
+/obj/item/cargo_backpack/Destroy()
+	QDEL_NULL(contained_packages)
+	return ..()
+
+/obj/item/cargo_backpack/examine(mob/user, distance)
+	. = ..()
+	if(length(contained_packages))
+		for(var/obj/item/cargo_package/package in contained_packages)
+			to_chat(user, SPAN_NOTICE(" - SITE: <b>[package.delivery_point_sector]</b> | COORD: <b>[package.delivery_point_coordinates]</b> | ID: <b>[package.delivery_point_id]</b>"))
```

Comment:
may be worth making this a small tgui opened on examine

Reply:
added, looks like this:
![image](https://github.com/Aurorastation/Aurora.3/assets/22774890/8f5eeedf-1be2-464c-87fc-a1cc97de16e4)